.github/workflows/steam.yml

```diff
@@ -0,0 +1,214 @@
+name: Steam Upload
+
+on:
+  release:
+    types:
+    - published
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to fetch and upload (nightly if none)'
+        required: false
+      win_url_override:
+        description: 'Windows build to use (.zip only)'
+        required: false
+      mac_url_override:
+        description: 'Mac build to use (.dmg only)'
+        required: false
+  schedule:
+  - cron: 0 0 * * *
```

Comment:
Do we want this to run every 24 hours or every 12 hours? The latter can be achieved with this:
```
0 */12 * * *
```

I don't have a strong opinion here.

Reply:
I went with 24 hours so it's an actualy "night"-ly :P